extensions/harvest/src/listTimeEntries.tsx

```diff
@@ -134,36 +364,26 @@ function DeleteEntryAction({
   onComplete: () => Promise<void>;
   entry: HarvestTimeEntry;
 }) {
-  return (
-    <PushAction
-      target={<Delete onComplete={onComplete} entry={entry} />}
-      title="Delete Time Entry"
-      shortcut={{ key: "delete", modifiers: ["cmd"] }}
-      icon={Icon.Trash}
-    />
-  );
-}
-
-function ToggleTimerAction({ entry, onComplete }: { entry: HarvestTimeEntry; onComplete: () => Promise<void> }) {
   return (
     <ActionPanelItem
-      title={entry.is_running ? "Stop Timer" : "Start Timer"}
-      icon={Icon.Clock}
       onAction={async () => {
-        const toast = new Toast({ style: ToastStyle.Animated, title: "Loading..." });
-        await toast.show();
-        try {
-          if (entry.is_running) {
-            await stopTimer(entry);
-          } else {
-            await restartTimer(entry);
-          }
-          await toast.hide();
-        } catch {
-          await showToast(ToastStyle.Failure, "Error", `Could not ${entry.is_running ? "stop" : "start"} your timer`);
-        }
-        await onComplete();
+        await confirmAlert({
+          title: "Delete Time Entry",
+          message: "Are you sure? This cannot be undone",
+          icon: Icon.Trash,
+          primaryAction: {
+            title: "Delete",
+            style: AlertActionStyle.Destructive,
+            onAction: async () => {
+              await deleteTimeEntry(entry);
+              await onComplete();
+            },
+          },
+        });
```

Comment:
while you can register a handler for an action, it's more elegant to use the `if (await confirmAlert(...)) { ... }` pattern
```suggestion
        if (await confirmAlert({
          title: "Delete Time Entry",
          message: "Are you sure? This cannot be undone",
          icon: Icon.Trash,
          primaryAction: {
            title: "Delete",
            style: AlertActionStyle.Destructive,
          },
        })) {
          await deleteTimeEntry(entry);
          await onComplete();
        }
```

I'll update the docs to clarify that